snare/tests/test_parse_timeout.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+from snare.utils.snare_helpers import parse_timeout
+
+
+class TestParseTimeout(unittest.TestCase):
+
+    def test_parse_timeout(self):
+        assert parse_timeout('20H') == 72000
```

Comment:
Can you write it as `20*60*60`? It will make more sense. And update the others too.

Reply:
Yes, makes more sense. Will do the changes